### DISCUSSION TITLE
The status on the Windows X2 machine does not change from queued.

### DISCUSSION THREAD

**Discussion by aitaro** [codemagic-ci-cd/codemagic-docs#2575] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. build flutter app on Windows Machine

### Expected results

A build starts.

### Actual results

A build not starting. Also, I cannot find intercom for getting support.

### Build id (optional)

665ef1a65cf3872cfea3d6d0

**Comment by helinanever** [comment]
Hi @aitaro, thank you for reporting this. We are aware of the issue and are currently investigating.

**Comment by helinanever** [comment, marked as answer]
The issue has been resolved and all builds are expected to start soon.
